### PR TITLE
Fix links to work in hugo rendered documentation, add additional stat…

### PR DESCRIPTION
**Pull request by thetechnick**
…ic types e.g. bool, int64

**What this PR does / why we need it**:
Links where not working and some types did not have links at all.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: hack/docgen/main.go [hunk 1]
```diff
@@ -304,8 +313,14 @@ func printAPIDocs(paths []string, sectionLink string) {
 		fmt.Println("| Field | Description | Scheme | Required |")
 		fmt.Println("| ----- | ----------- | ------ | -------- |")
 		fields := t[1:]
+
 		for _, f := range fields {
-			fmt.Println("|", f.Name, "|", f.Doc, "|", f.Type, "|", f.Mandatory, "|")
+			scheme := f.Type
+			// if !strings.HasPrefix(scheme, "[") {
```
Comment (aborilov) [inline hack/docgen/main.go:319]: do we need this commented piece of code?

Comment (thetechnick) [reply hack/docgen/main.go]: nope, sorry - fixed

**Pull request merged**